### PR TITLE
Add package pytmc

### PR DESCRIPTION
**Pull request by klauer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

To add @n-wbrown and @zllentz as maintainers later.

cc @hhslepicka

### CODE REVIEW

**Comment by hhslepicka** [comment]
I accept to be a maintainer of this recipe

**Comment by hhslepicka** [comment]
@conda-forge/help-python this is ready to go. Windows is failing due to entry_point script but all is good.

File: recipes/pytmc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
```
Comment (scopatz) [inline recipes/pytmc/meta.yaml]: please remove setuptools. You only need python and pip

Comment (klauer) [reply recipes/pytmc/meta.yaml]: Done

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**